src/csharp/Grpc.Examples/MathGrpc.cs

```diff
@@ -27,10 +27,35 @@ public static partial class Math
   {
     static readonly string __ServiceName = "math.Math";
 
-    static readonly grpc::Marshaller<global::Math.DivArgs> __Marshaller_math_DivArgs = grpc::Marshallers.Create((arg) => global::Google.Protobuf.MessageExtensions.ToByteArray(arg), global::Math.DivArgs.Parser.ParseFrom);
-    static readonly grpc::Marshaller<global::Math.DivReply> __Marshaller_math_DivReply = grpc::Marshallers.Create((arg) => global::Google.Protobuf.MessageExtensions.ToByteArray(arg), global::Math.DivReply.Parser.ParseFrom);
-    static readonly grpc::Marshaller<global::Math.FibArgs> __Marshaller_math_FibArgs = grpc::Marshallers.Create((arg) => global::Google.Protobuf.MessageExtensions.ToByteArray(arg), global::Math.FibArgs.Parser.ParseFrom);
-    static readonly grpc::Marshaller<global::Math.Num> __Marshaller_math_Num = grpc::Marshallers.Create((arg) => global::Google.Protobuf.MessageExtensions.ToByteArray(arg), global::Math.Num.Parser.ParseFrom);
+    static void __Helper_WriteBufferMessage(global::Google.Protobuf.IMessage message, global::Grpc.Core.SerializationContext context)
```

Comment:
nit: this will be passed to the Marshaller.Create as "serializer", so we might as well call the method "__Helper_SerializeMessage" (and __Helper_DeserializeMessage").
The "*Buffer*" in the name is unnecessary because the caller doesn't care if it's a buffer message or not.

Reply:
Good point. The methods are no longer specific to buffer messages.